test/data/extension-compat-test-extension.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Intentionally not private, useful if we want to re-generate the crx file for tests after doing changes.